### DISCUSSION TITLE
<Hashing template>

### DISCUSSION THREAD

**Discussion by adnan2770** [community/community#129319] [Programming Help]
### Select Topic Area

Question

### Body

Need a template for cp on hashing.

**Comment by Flash2over** [comment]
Maybe this Link helps: [https://www.geeksforgeeks.org/hashing-in-competitive-programming/](url)

**Reply by ebndev** [reply]
Hi @Flash2over, thanks for taking the time to answer!

@adnan2770 if the response above helped, please feel encouraged to mark it as the answer. If not, let us know if you have further questions or if you've found a template that worked for you. It would be helpful for other users who might experience this in the future 😄
